packages/webview_flutter/webview_flutter_android/example/lib/main.dart

```diff
@@ -23,10 +23,10 @@ const String kNavigationExamplePage = '''
 <head><title>Navigation Delegate Example</title></head>
 <body>
 <p>
-The navigation delegate is set to block navigation to the youtube website.
+The navigation delegate is set to block navigation to the pub.dev website.
```

Comment:
This should stay youtube since the test navigation delegate is blocking youtube.

```suggestion
The navigation delegate is set to block navigation to the youtube website.
```

Reply:
It isn't as far as I could see. 
https://github.com/flutter/packages/blob/d1d3bd6163977bd15417918b942fc18c221aa229/packages/webview_flutter/webview_flutter_android/example/lib/main.dart#L192

That's actually the reason I changed it, because the delegate didn't match the example.

Reply:
Ah, you're right. It looks like the url was changed in https://github.com/flutter/packages/pull/3879, but the example html wasn't. Thanks for catching this!